tests/docker/redis-ssl.dockerfile

```
@@ -0,0 +1,9 @@
+FROM redis:7.0-alpine
+COPY tests/ssl/redis/redis.crt /tmp.ssl/
+COPY tests/ssl/redis/redis.key /tmp.ssl/
+COPY tests/ssl/ca/ca.crt /tmp.ssl/ca.crt
+RUN chown -R redis:redis /tmp.ssl
+
+COPY docker/_scripts/redis/redis.conf /usr/local/etc/redis/redis.conf
+
+CMD ["redis-server", "/usr/local/etc/redis/redis.conf"]
```

```suggestion
ENTRYPOINT ["redis-server"]
CMD ["/usr/local/etc/redis/redis.conf"]
```
I'd suggest to use `ENTRYPOINT` command for a "base" part of command which pretend to be unchanged in most cases. [This makes it more natural changing container's behaviours during start](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint).